_posts/2021-03-12-february-rubygems-update.md

```diff
@@ -0,0 +1,36 @@
+---
+title: February 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in February.
+
+## RubyGems News
+
+This month on RubyGems, we fixed an issue about source priority that received attention due to a popular blog post (link to the blog post?).
+
+As always, we continue to fix bugs, review and merge PR’s and reply to support tickets.
+
+In February, [RubyGems](https://github.com/rubygems/rubygems) gained [114 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2021-02-01%7D...master@%7B2021-02-28%7D), contributed by 12 authors. There were 743 additions and 204 deletions across 56 files.
+
+## RubyGems.org News
+
+This month on RubyGems.org, we reduced the ngix rate limit to mitigate the load on Postgres - [#2634]( https://github.com/rubygems/rubygems.org/pull/2634) and evaluated the impact of reducing `random_page_post` in Postgres config.
+
+In addition to that, we made the following improvement and fixes:
+
+- updated elastic search to **v6.8**.
```

Comment:
Thanks to simi, we significantly improved the delay in our build time by migrating from Travis to Github Actions https://github.com/rubygems/rubygems.org/pull/2625

Reply:
Nice I'll include this.